### DISCUSSION TITLE
What's the release timeframe for DocFx 3.0?

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [dotnet/docfx#7521] [General]
I've been seeing a lot of exciting features that we would very much like to use in our DocFx projects going into the 3.0 branches, but I'm not sure how we should structure our plans for adopting those or not. Do you have any rough idea that you can share as to when DocFx 3.0 will be released?

Thank you for a great project!

**Comment by lanthonyneville** [comment]
Also keen to know. This is the latest update that I have seen: #7050

**Reply by Aaronontheweb** [reply]
Ah that's helpful: https://github.com/dotnet/docfx/blob/v3/docs/roadmap.md

**Comment by FSharpCSharp** [comment]
I can't wait for the new release either. So many great new features have been announced. I'm curious when the items on the roadmap will be addressed or completed. It was said there that this should be in late summer. According to the latest feedback, that should be the case soon.
